CoroutineTemplate/domain/build.gradle

```diff
@@ -1,41 +1,19 @@
-apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-parcelize'
-
-android {
-    compileSdkVersion android_compile_sdk_version
-    defaultConfig {
-        minSdkVersion android_min_sdk_version
-        targetSdkVersion android_target_sdk_version
-        versionCode android_version_code
-        versionName android_version_name
-
-        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
-        consumerProguardFiles "consumer-rules.pro"
-    }
+plugins {
+    id 'java-library'
+    id 'kotlin'
+}
 
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
-    kotlinOptions {
-        jvmTarget = JavaVersion.VERSION_1_8
-    }
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
 }
 
 dependencies {
     implementation(
-        project(':data'),
-
-        "androidx.core:core-ktx:$androidx_core_ktx_version",
-        "androidx.security:security-crypto:$android_crypto_version",
-        "com.google.dagger:hilt-android:$hilt_version",
-        "org.jetbrains.kotlinx:kotlinx-coroutines-android:$kotlinx_coroutines_version"
+        "com.squareup.moshi:moshi-kotlin:$moshi_version",
+        "com.squareup.retrofit2:converter-moshi:$retrofit_version",
+        "com.squareup.moshi:moshi:$moshi_version",
```

Comment:
These dependencies should be in the `data` module since all the networking and storage-related stuff is the responsibility of `data`.

Reply:
@AVI5HEK Actually, they are adapter for converter factories, you can use them for many purposes. In the current implementation, we're putting the `Response at `domain`, hence, it depends on our last decision to keep the `Response` or not in `domain`